main.yaml

```diff
@@ -74,6 +74,10 @@ Parameters:
     MaxValue: 16000
   ###############################################################################
   # Chef Server Stack Settings
+  ChefServerCustomConfig:
+    Default: ''
+    Type: String
+    Description: The S3 location of a custom chef-server configuration snippet that will be injected into the chef-server.rb (optional)
```

Comment:
hey @jeffj254 could you please move this to the bottom of this section?   In terms of cognitive burden we want optional things like these to be lower down.

Reply:
Moved!